docs/release-notes/1.8.3.md

```diff
@@ -0,0 +1,3 @@
+## New features
+
+- Change certificate key size to 4096 bits [#984](https://github.com/kyma-project/istio/pull/984)
```

Comment:
```suggestion
- The self-signed CA certificate's bit length is now set to `4096` instead of the default `2048`. [#984](https://github.com/kyma-project/istio/pull/984)
```